components/discord_webhook/actions/send-message-advanced/send-message-advanced.mjs

```diff
@@ -46,13 +46,21 @@ export default {
       content += `\n\n${this.getSentViaPipedreamText()}`;
     }
 
-    // No interesting data is returned from Discord
-    await this.discordWebhook.sendMessage({
+    const params = {
       avatarURL,
       embeds,
       content,
       threadID,
       username,
-    });
+    };
+
+    try {
+      // No interesting data is returned from Discord
+      await this.discordWebhook.sendMessage(params);
+      $.export("$summary", "Message sent successfully");
+    } catch (err) {
+      $.export("$summary", `${err}`);
+      $.export("unsent", params);
```

Comment:
Should the data for the `unsent` export be the user-configured prop values rather than the `params` object? If the `includeSentViaPipedream` prop is `true`, `content` is different from `this.message`. Emitting an object with the prop values instead would allow users to resend the message using those prop values.

Reply:
Addressed